src/main.cpp

```diff
@@ -160,7 +161,7 @@ void loop() {
   }
 
   // auto sleep on first screen
-  if (appPtr == 0 && (millis() - appOnScreenSince) > 5000) {
+  if (appPtr == (0 || 1) && (millis() - appOnScreenSince) > 10000) {
```

Comment:
(0 || 1) evaluates to true (1). You want something like:

`if ((appPtr == 0 || appPtr == 1) && (millis() - appOnScreenSince) > 10000)`